extensions/bed-time-calculator/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "bed-time-calculator",
+  "title": "Bed Time Calculator",
+  "description": "This is a simple extension for Raycast that will help you calculate the best time to sleep or wake up.",
+  "icon": "extension_icon.png",
+  "author": "khlebobul",
+  "categories": [
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "sleep-now",
+      "title": "Sleep Now",
+      "description": "I'm going to sleep right now and want to know when to wake up",
+      "mode": "view"
+    },
+    {
+      "name": "sleep-at",
+      "title": "Sleep At",
+      "description": "I know when I'll fall asleep and want to know when to wake up",
+      "mode": "view"
+    },
+    {
+      "name": "wake-up-at",
+      "title": "Wake Up At",
```

Comment:
```suggestion
      "title": "Wake up At",
```

Reply:
Good afternoon, you are right, that would be better. I will definitely familiarise myself with how to do this and send it for checking again, thanks for the recommendations.

Reply:
Awesome 🔥